media-libs/libyuv/files/0001-fix-install-dirs-1909.patch

```diff
@@ -0,0 +1,43 @@
+From 9fc18452acdba9a20a8ffb6036c081f937b3b95e Mon Sep 17 00:00:00 2001
+From: NRK <nrk@disroot.org>
+Date: Thu, 1 May 2025 06:58:03 +0000
+Subject: [PATCH] fix install dirs
+
+---
+ CMakeLists.txt | 11 ++++++-----
+ 1 file changed, 6 insertions(+), 5 deletions(-)
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 16702d3..ec87d94 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -2,9 +2,10 @@
+ # Originally created for "roxlu build system" to compile libyuv on windows
+ # Run with -DUNIT_TEST=ON to build unit tests
+ 
+-cmake_minimum_required( VERSION 3.16 )
++cmake_minimum_required( VERSION 3.2 )
```

Comment:
Uhh, just noticed that this change was probably unnecessary. I made it since I thought `3.2` is higher than `3.16` (in fractional number). But that's not the case with version numbers. Will fix soon.

Reply:
done